.env.test

```diff
@@ -0,0 +1,14 @@
+# Flask server configuration.
+FLASK_APP=app
+FLASK_ENV=development
+FLASK_SKIP_DOTENV=True
+
+# Change the following values for local development.
+LOGIN_DISABLED=False
+SECRET_KEY=secret-key
+API_KEY=api123
+TOKEN=token123
+BOARD_ID=board123
+TODO_LIST_ID=456
+MONGO_DB_CONNECTION=mongodb+srv://rkorol1:Docker123@cluster0.1pfca.mongodb.net/myFirstDatabase?w=majority
```

Comment:
These are real credentials, please change the password ASAP. Your integration tests shouldn't need a real database and your e2e tests should get this from the environment or load the real `.env` file